src/core/MouseCapture.tsx

```diff
@@ -89,7 +93,32 @@ export default class MouseCapture extends React.Component<Props, State> {
     });
   }
 
-  private _onMouseDown = (e: React.MouseEvent) => {
+  private _maybeDispatchDragHandler(
+    e: React.MouseEvent | MouseEvent,
+    handler: (xPct: number, yPct: number, e: React.MouseEvent | MouseEvent) => void
+  ) {
+    if (e.button === LEFT_MOUSE_BUTTON && handler && this.state.mouseDownClientX != null) {
+      const { xScale, yScale } = this._createPhysicalToLogicalScales();
+      const { left, right, top, bottom } = this.element.getBoundingClientRect();
+      handler(
+        xScale(Math.min(Math.max(e.clientX, left), right)),
```

Comment:
What caused you to decide to limit it to the area of the element here? It could be nice if you could drag into areas that aren't visible, for consistency.